client-app/src/api/auth.js

```diff
@@ -1,22 +1,24 @@
-import axios from "./axios"
-
+import axios from "axios";
 export default class AuthApi {
-
-    static async register(fullName, email, password) {
-        try {
-            const {data} = await axios.post("/auth/register", {fullName, email, password})
-            return data
-        } catch (e) {
-            console.log(e)
-        }
+  static async register(fullName, email, password) {
+    try {
+      const { data } = await axios.post("api/auth/register", {
+        fullName,
+        email,
+        password,
+      });
+      return data;
+    } catch (e) {
+      console.log(e);
     }
+  }
```

Comment:
Why those changes? I think we have different formatters, this can make our life difficult